luigi/contrib/hdfs/__init__.py

```diff
@@ -19,856 +19,58 @@
 Provides access to HDFS using the :py:class:`HdfsTarget`, a subclass of :py:class:`~luigi.target.Target`.
 You can configure what client by setting the "client" config under the "hdfs" section in the configuration, or using the ``--hdfs-client`` command line option.
 "hadoopcli" is the slowest, but should work out of the box. "snakebite" is the fastest, but requires Snakebite to be installed.
-"""
-
-import datetime
-import logging
-import os
-import random
-import re
-import subprocess
-try:
-    from urlparse import urlsplit
-except ImportError:
-    from urllib.parse import urlsplit
-import warnings
 
-from luigi import six
+Currently (4th May) the :py:mod:`luigi.contrib.hdfs` module is under
+reorganization. We recommend importing the reexports from
+:py:mod:`luigi.contrib.hdfs` instead of the sub-modules, as we're not yet sure
+how the final structure of the sub-modules will be yet. Eventually this module
+will be empty and you'll have to import directly from the sub modules like
+:py:mod:`luigi.contrib.hdfs.config`.
+"""
 
-import luigi.contrib.target
-import luigi.format
-from luigi.target import FileAlreadyExists, FileSystem, FileSystemTarget
+# Here follows a series of deprecated imports that have been moved out there own files.
```

Comment:
Confusing comment, unneeded.
* It looks more like saying that the contrib style of imports is deprecated, but that's the new shiny style of imports, right?
* Everything phrased as "here follows" is prone to become outdated because people who contribute to something far below in this code can't be expected to notice the comment at the top and add a "here the series of deprecated imports end. Here follow my lovely imports unrelated to that refactor".